vendor/github.com/prometheus/exporter-toolkit/web/tls_config.go

```diff
@@ -321,16 +321,16 @@ func (c *curve) MarshalYAML() (interface{}, error) {
 	return fmt.Sprintf("%v", c), nil
 }
 
-type tlsVersion uint16
+type TLSVersion uint16
 
-var tlsVersions = map[string]tlsVersion{
-	"TLS13": (tlsVersion)(tls.VersionTLS13),
-	"TLS12": (tlsVersion)(tls.VersionTLS12),
-	"TLS11": (tlsVersion)(tls.VersionTLS11),
-	"TLS10": (tlsVersion)(tls.VersionTLS10),
+var tlsVersions = map[string]TLSVersion{
+	"TLS13": (TLSVersion)(tls.VersionTLS13),
+	"TLS12": (TLSVersion)(tls.VersionTLS12),
+	"TLS11": (TLSVersion)(tls.VersionTLS11),
+	"TLS10": (TLSVersion)(tls.VersionTLS10),
```

Comment:
It seems to me that we accept different strings in YAML ("TLS13") and CLI flag ("VersionTLS13"). That seems like a very bad idea to me, and it will be confusing for users.

Reply:
Where do we accept those strings in YAML?

Reply:
> Where do we accept those strings in YAML?

`(*TLSVersion).UnmarshalYAML` uses `tlsVersions` map to check if the value is correct, and this map has keys like "TLS13", "TLS10", etc. Am I reading it wrong?